atkinson/errors/dlrn.py

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env python
+"""
+Atkinson Dlrn Error Handling.
+
+The error classes here are for errors that occur when interacting with dlrn
+and not errors in the atkinson applicaiton itself.
+"""
+import datetime
+import os.path
+
+from atkinson.errors import BaseError
+
+
+class DlrnFTBFSError(BaseError):
```

Comment:
Please add some docstrings to this.

Reply:
What type of docstring are you looking for her beyond what is on the methods? That class has a docstring on the line below.

Reply:
Ok, I guess I really just mean the init, specifically the valid params.  I had to read a little ways through the code before it was clear to me what was expected to be passed in for reporter, for instance.

Reply:
Ah, fair. I will certainly add those.